spec/mail/body_spec.rb

```diff
@@ -443,7 +443,7 @@ def assert_split_into(body, pre, epi, parts)
 
   describe "non US-ASCII charset" do
     it "should encoded" do
-      body = Mail::Body.new("あいうえお\n")
+      body = Mail::Body.new("あいうえお\r\n")
```

Comment:
Since non-ASCII chars are seen as possibly-binary. We could improve the heuristic to check for other valid text encodings to allow this.